File: src/common/helpers/userHelper.ts
import { Roles } from "../enums/Roles";

export const getRoleById = (roleId: number): Roles => {
  switch (roleId) {
    case 1:
      return Roles.Admin;
    case 2:
      return Roles.Manager;
    case 3:
      return Roles.User;
    default:
      return Roles.User;
  }
};

File: TwojUrlop.DomainModel/Common/Helpers/UserHelper.cs
using System;
using System.Collections.Generic;
using Entities = TwojUrlop.Common.Models.Entities;
using Enums = TwojUrlop.Common.Enums;

namespace TwojUrlop.DomainModel.Common.Helpers;
public class UserHelper
{
    public static bool IsManagerOrAdmin(Entities.User user)
    {
        return user.RoleId == (int)Enums.Role.Manager || user.RoleId == (int)Enums.Role.Admin;
    }
}